src/utils/OWLConversor/OWLConversor.js

```diff
@@ -0,0 +1,145 @@
+import './xml2json.js';
+import './Utils.js';
+import './Query.js';
+
+(function(){
+	window.MAX_OBJECTS_PER_ROW = 5;
+	
+	var x2js = new X2JS({attributePrefix :"@"});
+	
+	var Config = require("./config.terms.json");
```

Comment:
Don't use `require`, use `import` instead and do it on the top's file.

```js
import config from './config.terms.json';
```

Camelocase:
`Config ` -> `config`
`IgnoreList` -> `ignoreList`
...